codalab_service.py

```diff
@@ -802,16 +585,10 @@ def start_services(self):
         if should_run_service(self.args, 'test'):
             print_header('Running tests')
             self.run_service_cmd(
-                "/opt/wait-for-it.sh rest-server:2900 -- python test_cli.py --instance http://rest-server:2900 default",
-                root=(not self.args.codalab_home),
+                wait_rest_server('python test_cli.py --instance {} default'.format(rest_url))
             )
 
-        if should_run_service(self.args, 'monitor'):
-            print_header('Running monitoring service')
-            self.run_service_cmd(
-                "/opt/wait-for-it.sh rest-server:2900 -- python monitor.py",
-                root=(not self.args.codalab_home),
-            )
+        self.bring_up_service('monitor')
```

Comment:
It seems like `monitor` is being run by default?

Reply:
no, the check is done inside `bring_up_service`

Reply:
Got it, thanks